server/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
```

Comment:
Put imports at the top - `const cors = require('cors');` goes before `const app = express()`.